src/libs/Navigation/linkingConfig/FULL_SCREEN_TO_RHP_MAPPING.ts

```diff
@@ -74,7 +74,13 @@ const FULL_SCREEN_TO_RHP_MAPPING: Partial<Record<FullScreenName, string[]>> = {
         SCREENS.WORKSPACE.TAG_EDIT,
         SCREENS.WORKSPACE.TAG_LIST_VIEW,
     ],
-    [SCREENS.WORKSPACE.CATEGORIES]: [SCREENS.WORKSPACE.CATEGORY_CREATE, SCREENS.WORKSPACE.CATEGORY_SETTINGS, SCREENS.WORKSPACE.CATEGORIES_SETTINGS, SCREENS.WORKSPACE.CATEGORY_EDIT],
+    [SCREENS.WORKSPACE.CATEGORIES]: [
+        SCREENS.WORKSPACE.CATEGORY_CREATE,
+        SCREENS.WORKSPACE.CATEGORY_SETTINGS,
+        SCREENS.WORKSPACE.CATEGORIES_SETTINGS,
+        SCREENS.WORKSPACE.CATEGORY_EDIT,
+        SCREENS.WORKSPACE.CATEGORY_GLCODE,
```

Comment:
mayyyyy want to call this `_EDIT_GLCODE`? Maybe?

Reply:
yeah sure, i'll update this everywhere actually.